plugins/obs-outputs/CMakeLists.txt

```diff
@@ -1,200 +1,184 @@
+          "OBS: mbedTLS or zlib was not found, RTMPS will be auto-disabled")
+      target_compile_definitions(obs-outputs PRIVATE NO_CRYPTO)
+    endif()
+  else()
+    target_sources(
+      obs-outputs
+      PRIVATE librtmp/amf.c
+              librtmp/amf.h
+              librtmp/bytes.h
+              librtmp/cencode.c
+              librtmp/cencode.h
+              librtmp/dh.h
+              librtmp/dhgroups.h
+              librtmp/handshake.h
+              librtmp/hashswf.c
+              librtmp/http.h
+              librtmp/log.c
+              librtmp/log.h
+              librtmp/md5.c
+              librtmp/md5.h
+              librtmp/parseurl.c
+              librtmp/rtmp.c
+              librtmp/rtmp.h
+              librtmp/rtmp_sys.h)
+
+    target_compile_definitions(obs-outputs PRIVATE USE_MBEDTLS CRYPTO)
+
+    target_link_libraries(obs-outputs PRIVATE Mbedtls::Mbedtls ZLIB::ZLIB)
+
+    if(OS_WINDOWS)
+      target_link_libraries(obs-outputs PRIVATE crypt32)
+
+    elseif(OS_MACOS)
+      find_library(FOUNDATION_FRAMEWORK Foundation)
+      find_library(SECURITY_FRAMEWORK Security)
+      mark_as_advanced(FOUNDATION_FRAMEWORK SECURITY_FRAMEWORK)
+
+      target_link_libraries(obs-outputs PRIVATE ${FOUNDATION_FRAMEWORK}
+                                                ${SECURITY_FRAMEWORK})
+      set_target_properties(obs-outputs PROPERTIES CXX_VISIBILITY_PRESET hidden)
+
+    elseif(OS_POSIX)
+      set_target_properties(obs-outputs PROPERTIES CXX_VISIBILITY_PRESET hidden)
+    endif()
+  endif()
+else()
+  target_compile_definitions(obs-outputs PRIVATE NO_CRYPTO)
 endif()
 
-set(obs-outputs_HEADERS
-	"${CMAKE_BINARY_DIR}/plugins/obs-outputs/config/obs-outputs-config.h"
-	obs-output-ver.h
-	rtmp-helpers.h
-	rtmp-stream.h
-	net-if.h
-	flv-mux.h)
-set(obs-outputs_SOURCES
-	obs-outputs.c
-	null-output.c
-	rtmp-stream.c
-	rtmp-windows.c
-	flv-output.c
-	flv-mux.c
-	net-if.c)
-
-if(WIN32)
-	set(MODULE_DESCRIPTION "OBS output module")
-	configure_file(${CMAKE_SOURCE_DIR}/cmake/winrc/obs-module.rc.in obs-outputs.rc)
-	list(APPEND obs-outputs_SOURCES
-		obs-outputs.rc)
+find_package(PkgConfig QUIET)
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(FTL QUIET libftl)
 endif()
 
-add_library(obs-outputs MODULE
-	${ftl_SOURCES}
-	${ftl_HEADERS}
-	${obs-outputs_SOURCES}
-	${obs-outputs_HEADERS}
-	${obs-outputs_librtmp_SOURCES}
-	${obs-outputs_librtmp_HEADERS})
-
 if(FTL_FOUND)
-	target_link_libraries(obs-outputs ${FTL_LIBRARIES})
-	target_include_directories(obs-outputs PUBLIC ${FTL_INCLUDE_DIRS})
+  find_package(CURL REQUIRED)
+  message(STATUS "OBS:   found ftl-sdk (system): ftl outputs enabled")
+
+  target_sources(obs-outputs PRIVATE ftl-stream.c)
+
+  target_include_directories(obs-outputs PRIVATE ${FTL_INCLUDE_DIRS})
+
+  target_link_libraries(obs-outputs PRIVATE ${FTL_LIBRARIES} CURL::libcurl)
+
+  target_compile_features(obs-outputs PRIVATE c_std_11)
+
+elseif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/ftl-sdk/CMakeLists.txt")
```

Comment:
`target_compile_definitions(obs-outputs PRIVATE FTL_FOUND)` is not present when FTL is found on the system.

Reply:
Correct, fixed.